tests/test_modeling_albert.py

```diff
@@ -227,6 +230,20 @@ class AlbertModelTest(ModelTesterMixin, unittest.TestCase):
         else ()
     )
 
+    # special case for ForPreTraining model
+    def _prepare_for_class(self, inputs_dict, model_class, return_labels=False):
+        inputs_dict = super()._prepare_for_class(inputs_dict, model_class, return_labels=return_labels)
```

Comment:
since tests are not user-facing, I decided to use inheritance here instead of overwriting. Is that fine?

Reply:
No problem with this on my side.